uisdk/src/main/res/layout/uisdk_view_quotes_item_prebook.xml

```diff
@@ -0,0 +1,177 @@
+                        android:layout_gravity="center" />
+
+                    <ImageView
+                        android:id="@+id/logoImage"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:layout_gravity="center"
+                        android:scaleType="fitCenter" />
+
+                    <ImageView
+                        android:id="@+id/loadingIcon"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:layout_gravity="center"
+                        android:scaleType="centerInside"
+                        app:srcCompat="@drawable/uisdk_ic_quotes_logo_loading" />
+
+                </FrameLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:layout_marginStart="@dimen/kh_uisdk_spacing_semi_small"
+                    android:orientation="vertical"
+                    android:paddingEnd="@dimen/kh_uisdk_spacing_x6small">
+
+                    <TextView
+                        android:id="@+id/categoryText"
+                        style="@style/QuotesFontTitleTextAppearance"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="left"
+                        android:ellipsize="end"
+                        android:maxLines="1"
+                        android:textSize="@dimen/kh_uisdk_text_size_small_medium"
+                        tools:text="Car" />
+
+                    <com.karhoo.uisdk.screen.booking.quotes.capacity.CapacityView
+                        android:id="@+id/capacityWidget"
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent" />
+
+
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:gravity="center_vertical|end"
+
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/priceText"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="center_vertical|end"
+                        style="@style/QuotesFontTextAppearance"
+                        android:textStyle="bold"
+                        android:maxLines="1"
+                        android:textColor="@color/kh_uisdk_text_primary"
+                        android:textSize="@dimen/kh_uisdk_text_size_large"
+                        tools:text="GBP100.00 - 100.00" />
```

Comment:
Do we need this hardcoded value?

Reply:
tools:text is used only for design preview for us to see how it will look like, it will never be the actual text in the control, that's android:text